tck/src/main/resources/tests/basicScalarTests.csv

```diff
@@ -1,25 +1,53 @@
-type ScalarHolder   |   longObject: Long                                        |   Expecting a Long Scalar Type
-type ScalarHolder   |   longPrimitive: Long                                     |   Expecting a Long Scalar Type
-type ScalarHolder   |   shortObject: Short                                      |   Expecting a Short Scalar Type
-type ScalarHolder   |   shortPrimitive: Short                                   |   Expecting a Short Scalar Type
-type ScalarHolder   |   stringObject: String                                    |   Expecting a String Scalar Type
-type ScalarHolder   |   stringPrimitive: String                                 |   Expecting a String Scalar Type
-type ScalarHolder   |   timeObject: Time                                        |   Expecting a Time Scalar Type 
\ No newline at end of file
+1|  type ScalarHolder   |   bigDecimalObject: BigDecimal                            |   Expecting a BigDecimal Scalar Type in type ScalarHolder
+2|  type ScalarHolder   |   bigIntegerObject: BigInteger                            |   Expecting a BigInteger Scalar Type in type ScalarHolder
+3|  type ScalarHolder   |   booleanObject: Boolean                                  |   Expecting a Boolean Scalar Type in type ScalarHolder
+4|  type ScalarHolder   |   booleanPrimitive: Boolean!                              |   Expecting a non null Boolean Scalar Type in type ScalarHolder
+5|  type ScalarHolder   |   byteObject: Byte                                        |   Expecting a Byte Scalar Type in type ScalarHolder
+6|  type ScalarHolder   |   bytePrimitive: Byte!                                    |   Expecting a non null Byte Scalar Type in type ScalarHolder
+7|  type ScalarHolder   |   charObject: Char                                        |   Expecting a Char Scalar Type in type ScalarHolder
+8|  type ScalarHolder   |   charPrimitive: Char!                                    |   Expecting a non null Char Scalar Type in type ScalarHolder
+9|  type ScalarHolder   |   dateObject: Date                                        |   Expecting a Date Scalar Type in type ScalarHolder
+10| type ScalarHolder   |   dateTimeObject: DateTime                                |   Expecting a DateTime Scalar Type in type ScalarHolder
+11| type ScalarHolder   |   doubleObject: Float                                     |   Expecting a Float Scalar Type in type ScalarHolder
+12| type ScalarHolder   |   doublePrimitive: Float!                                 |   Expecting a non null Float Scalar Type in type ScalarHolder
+13| type ScalarHolder   |   floatObject: Float                                      |   Expecting a Float Scalar Type in type ScalarHolder
+14| type ScalarHolder   |   floatPrimitive: Float!                                  |   Expecting a non null Float Scalar Type in type ScalarHolder
+15| type ScalarHolder   |   id: ID                                                  |   Expecting a ID Scalar Type in type ScalarHolder
```

Comment:
This fails for me - I still get the same value as before `id: String`.  I think this either needs to be `id: String` or the `id` field in ScalarHolder needs to have the `@Id` annotation on it.

Reply:
Yes. My mistake. Added a `@Id` in the app. Will be included in the next commit.